homeassistant/components/overkiz/sensor.py

```diff
@@ -413,6 +413,14 @@ class OverkizSensorDescription(SensorEntityDescription):
         options=["open", "tilt", "closed"],
         translation_key="three_way_handle_direction",
     ),
+    # Hitachi air to air heatpump outdoor temperature sensors
+    OverkizSensorDescription(
+        key="hlrrwifi:OutdoorTemperatureState",
+        name="Outdoor temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+    ),
 ]
```

Comment:
Please remove this sensor and add this in a new PR. (+ use an enum for `hlrrwifi:OutdoorTemperatureState`).

Reply:
No strong objections. Out of curiosity, this sensor being part of the added device and fully tested, is your recommendation because of the string constant which is added in the [pyoverkiz 3.11](https://github.com/home-assistant/core/pull/101024) enums (thanks for that)?

Reply:
Mainly because one PR should target a single entity platform.